app/src/test/java/com/kks/nimblesurveyjetpackcompose/UnitTestConstants.kt

```diff
@@ -0,0 +1,38 @@
+package com.kks.nimblesurveyjetpackcompose
+
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType
+import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
+import com.kks.nimblesurveyjetpackcompose.model.SurveyQuestion
+
+val surveyQuestions = listOf(
+    SurveyQuestion(
+        id = "0",
+        title = "title question",
+        displayOrder = 0,
+        shortText = "",
+        pick = "none",
+        questionDisplayType = QuestionDisplayType.NONE,
+        answers = listOf(
+            SurveyAnswer(
+                id = "0",
+                text = "text",
+                displayOrder = 0
+            )
+        )
+    ),
+    SurveyQuestion(
+        id = "1",
+        title = "title",
+        displayOrder = 0,
+        shortText = "",
+        pick = "none",
+        questionDisplayType = QuestionDisplayType.NONE,
+        answers = listOf(
+            SurveyAnswer(
+                id = "0",
+                text = "text",
+                displayOrder = 0
+            )
+        )
+    )
```

Comment:
Same suggestion as on the other PR -  we can use a property and use `copy()` method to reduce code.